ArgusCore/src/test/java/com/salesforce/dva/argus/service/AlertServiceIT.java

```diff
@@ -99,7 +99,9 @@ public void testExecuteScheduledAlerts_ForOneTimeSeries() throws InterruptedExce
             for (int i = 0; i < expectedNotifications; i++) {
                 for (int j = 0; j < 1 + random.nextInt(5); j++) {
                     alertService.enqueueAlerts(Arrays.asList(new Alert[] { alert }));
+                    sleep(5000);
```

Comment:
@saaja-sfdc, I think this was added by @rajsarkapally-sfdc to make the integration tests pass.